eng/common/scripts/Queue-Pipeline.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$Organization,
+
+  [Parameter(Mandatory = $true)]
+  [string]$Project,
+
+  [Parameter(Mandatory = $true)]
+  [string]$SourceBranch,
+
+  [Parameter(Mandatory = $true)]
+  [int]$DefinitionId,
+
+  [Parameter(Mandatory = $true)]
+  [string]$AuthToken
+)
+
+. "${PSScriptRoot}\logging.ps1"
+
+$encodedAuthToken = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes([string]::Format("{0}:{1}", "", $AuthToken)))
+$headers = @{
+  Authorization = "Basic $encodedAuthToken"
+}
+
+$apiUrl = "https://dev.azure.com/$Organization/$Project/_apis/build/builds?api-version=6.0"
+
+$body = @{
+  sourceBranch = $SourceBranch
+  definition = @{ id = $DefinitionId }
+}
+
+Write-Verbose ($body | ConvertTo-Json)
+
+try {
+  $resp = Invoke-RestMethod -Method POST -Headers $headers $apiUrl -Body ($body | ConvertTo-Json) -ContentType application/json
```

Comment:
Please verify that in failure conditions that the ```$headers``` variable isn't output in any way since it contains a credential.